x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/draggable_dimension_button.tsx

```diff
@@ -105,6 +107,18 @@ export function DraggableDimensionButton({
     columnId,
   ]);
 
+  /* 2 to leave room for data panel and workspace, then go by layer index, then by group index */
+  const order = useMemo(() => [2, layerIndex, groupIndex, accessorIndex], [
```

Comment:
Not sure about the value here for the `useMemo`. It's not a heavy computation and the same memory used to declare the dependency is used to assign the value to the `order` value. I think it's ok to remove this `useMemo`